builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -432,7 +432,7 @@ func (b *backend) pathPatchIssuer(ctx context.Context, req *logical.Request, dat
 	}
 
 	// Leaf Not After Changes
-	rawLeafBehaviorData, ok := data.GetOk("leaf_not_after_behaivor")
+	rawLeafBehaviorData, ok := data.GetOk("leaf_not_after_behavior")
```

Comment:
While this is a typo, I don't think we can change this due to backwards compatibility issues?